projects/plugins/jetpack/tests/e2e/jest.config.js

```diff
@@ -9,9 +9,9 @@ if ( process.env.E2E_DEBUG ) {
 }
 
 module.exports = {
-	testEnvironment: '<rootDir>/lib/env/playwright-environment.js',
-	globalSetup: '<rootDir>/lib/env/global-setup.js',
-	globalTeardown: '<rootDir>/lib/env/global-teardown.js',
+	testEnvironment: '<rootDir>/node_modules/jetpack-e2e-commons/env/playwright-environment.js',
```

Comment:
Could these do something like
```suggestion
	testEnvironment: require.resolve( 'jetpack-e2e-commons/env/playwright-environment.js' ),
```
instead of explicitly digging into `node_modules`? Or does that not work in the jest config?

Reply:
Nice, it looks like it works. Solved with [d0dd564](https://github.com/Automattic/jetpack/pull/21115/commits/d0dd564db571414de0611a3697f8f8bfe4678888).